client/state/automated-transfer/selectors.js

```diff
@@ -48,3 +54,15 @@ export const getEligibility = compose(
 	getEligibilityData,
 	getAutomatedTransfer,
 );
+
+/**
+ * Checks if the site is currently transferring
+ *
+ * @param {Object} state global app state
+ * @param {number} siteId requested site for tranfer info
+ * @returns {bool} transferring check
+ */
+export const isAutomatedTransferTransferring = ( state, siteId ) => includes(
```

Comment:
I can't say I'm a fan of _this_ name I've chosen, but mostly for repeating "transfer".
Unfortunately, I can't think of a better term that's still as clear as this one, unless we replaced "AutomatedTransfer" with its acronym, but I don't think I've ever seen acronyms used anyway in Calypso - and also it would make sense to us internally (`:atat:`), but likely not to anybody else.